src/wordpress.js

```diff
@@ -48,7 +50,7 @@ const install = async function( env, envHost, answers ) {
             break;
     }
 
-    execSync( `docker-compose exec phpfpm wp core ${command} ${flags} --url=http://${envHost} --title="${answers.title}" --admin_user="${answers.username}" --admin_password="${answers.password}" --admin_email="${answers.email}"`, { stdio: 'inherit', cwd: envPath });
+    execSync( `docker-compose exec phpfpm wp core ${command} ${flags} --url=${http}://${envHost} --title="${answers.title}" --admin_user="${answers.username}" --admin_password="${answers.password}" --admin_email="${answers.email}"`, { stdio: 'inherit', cwd: envPath });
```

Comment:
@dustinrue it already does it. If SSL is enabled, then WordPress is installed as https://. See it here

Reply:
Right, but I'm saying everything else in the PR is not necessary as an SSL cert is shipped with wp-local-docker and is on at all times.